src/components/Calculator.js

```diff
@@ -1,31 +1,41 @@
-import React from 'react';
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
 
 function Calculator() {
+  const [state, setState] = useState({ total: 0, next: null, operation: null });
+
+  const clickHandler = (e) => {
+    setState((state) => calculate(state, e.target.value));
+  };
   return (
     <section>
       <div className="result">
-        <p>0</p>
+        <p>
+          {state.total}
+          {state.operation}
+          {state.next}
+        </p>
       </div>
       <div className="calculator">
-        <button type="button" className="button" value="AC">AC</button>
-        <button type="button" className="button" value="AC">+/-</button>
-        <button type="button" className="button" value="%">%</button>
-        <button type="button" className="operator" value="÷">÷</button>
-        <button type="button" className="button" value="7">7</button>
-        <button type="button" className="button" value="8">8</button>
-        <button type="button" className="button" value="9">9</button>
-        <button type="button" className="operator" value="*">*</button>
-        <button type="button" className="button" value="4">4</button>
-        <button type="button" className="button" value="5">5</button>
-        <button type="button" className="button" value="6">6</button>
-        <button type="button" className="operator" value="-"> -</button>
-        <button type="button" className="button" value="3">3</button>
-        <button type="button" className="button" value="2">2</button>
-        <button type="button" className="button" value="1">1</button>
-        <button type="button" className="operator" value="+">+</button>
-        <button type="button" className="button btn-0" value="0">0</button>
-        <button type="button" className="button" value=".">.</button>
-        <button type="button" className="operator" value="="> =</button>
+        <button type="button" className="button" onClick={clickHandler} value="AC">AC</button>
+        <button type="button" className="button" onClick={clickHandler} value="AC">+/-</button>
```

Comment:
- [ ] Currently the `+/-` symbol is acting like the `AC` because you are passing `AC` as a value instead of `+/-` even though it is displaying `+/-` on the user interface. Kindly update it to `value="+/-"` so it can work as expected. 😎️